DRILL-6797: Added tests to verify that split function no longer retur…

agozhiy opened the pull request:
…ns an error when result has no rows

Agirish commented:
Hey @agozhiy , I'm a bit confused here - I see a few new SQL files, and a few baseline files which have been renamed. Can you please clarify on the changeset?

agozhiy commented:
@Agirish, sorry, I added the following tests:
DRILL_6797_split_filters_combination.sql
DRILL_6797_split_having.sql
DRILL_6797_split_limit_0.sql
DRILL_6797_split_no_result.sql
DRILL_6797_split_with_result.sql
Also, I changed the extensions from .res to .e_tsv to meet the conventions.
